conf/messages

```diff
@@ -362,11 +362,11 @@ error.range.day=Day out of range
 error.range.month=Month out of range
 error.range.year=Year out of range
 error.date.invalid=Invalid date entered
-error.invalid.date.format=Enter a valid date
-error.enter_a_date=Enter a date
-head.error.enter_a_date=You need to enter a date
+error.invalid.date.format=Date you started to live here must be a real date
```

Comment:
Is that the right content?`real date` seems a bit off to me

Reply:
that's the content I got in the document attached to the ticket 
"Validation: date entered cannot be correct, for example 32/5/1990
 Current error: Check the date you entered and Enter a valid date
 New error: Date you started to live here must be a real date"

Reply:
yeah, that's the content from the audit document on the ticket 
"Validation: date entered cannot be correct, for example 32/5/1990 
Current error: Check the date you entered and Enter a valid date 
New error: Date you started to live here must be a real date "